libs/common-hooks/src/useAvailableChains.ts

```diff
@@ -3,13 +3,21 @@ import { useMemo } from 'react'
 import { getAvailableChains } from '@cowprotocol/common-utils'
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
-import { useFeatureFlags } from './index'
-
+/**
+ * Hook to get a list of SupportedChainId currently available/enabled
+ *
+ * Normally it is the exact same list of chains in the SupportChainId enum.
+ * It'll be different when a chain is behind a feature flag.
+ * Set the flag in this hook.
+ */
 export function useAvailableChains(): SupportedChainId[] {
-  const { isArbitrumOneEnabled } = useFeatureFlags()
+  // 1. Load feature flag for chain being enabled
+  // const { isArbitrumOneEnabled } = useFeatureFlags()
 
   return useMemo(
-    () => getAvailableChains(isArbitrumOneEnabled ? undefined : [SupportedChainId.ARBITRUM_ONE]),
-    [isArbitrumOneEnabled]
+    // 2. Conditionally build a list of chain ids to exclude
+    // () => getAvailableChains(isArbitrumOneEnabled ? undefined : [SupportedChainId.ARBITRUM_ONE]),
+    () => getAvailableChains(),
```

Comment:
why not just deleting all and re-exporting `getAvailableChains` under `useAvailableChains` name? 

Is it so you have this ready for the next chain?

Reply:
Exactly.
This hook is only useful when a chain that's part of the SDK shouldn't be displayed in the UI.
But touching everywhere when we need to hide it is cumbersome.
I rather have this "empty" wrapper in place to make the change simpler.